src/js/src/mixins/HistoryRoutingUtils.js

```diff
@@ -1,6 +1,8 @@
 export default {
   methods: {
     $_pushSearchHistory(destination, query, appliedFacets, solrSettings) {
+    // Should be deleted before production.
+    console.log('We pushed a history.')
```

Comment:
Remember to remove as the comment says

Reply:
leaving comments and console.logs for now.

Reply:
As agreed